trace/doc.go

```diff
@@ -46,8 +46,8 @@ lines to the top of the function:
     ctx = trace.StartSpan(ctx, "your choice of name")
     defer trace.EndSpan(ctx)
 
-StartSpan will create a child span if one already exists, and will create a
-new top-level span otherwise.
+Invoking StartSpan with the name of an already existent span will create a child span,
```

Comment:
s/already existent/existing/

Period after "child span".

Reply:
Thanks @jba, I'll update this in a second.